src/command.js

```diff
@@ -27,11 +29,29 @@ const Command = class {
 
   static create (rawInputs) {
     const command = doNotAllowMissingProperties(new this(rawInputs))
+    this._sanitizeCreation(command)
     this._validateCreation(command)
```

Comment:
This order matters. I found better results if I sanitized first and then validated.